src/test/java/it/gov/pagopa/pu/bff/security/WebSecurityConfigTest.java

```diff
@@ -24,6 +25,9 @@ class WebSecurityConfigTest {
   @MockBean
   private JwtAuthenticationFilter jwtAuthenticationFilter;
 
+  @MockBean
+  private BrokerServiceImpl service;
+
   @Test
   void givenURLWhenWithoutAccessTokenThenRedirectToLogin() throws Exception {
```

Comment:
This test name is asserting something that is not aligned with the implemented test...
It's testing the right exposition of actuators
(Moreover, BFF will never perform a redirect, at least it would return 401 or 403)

Reply:
fixed